### PR TITLE
Add mask and data props to input fields

### PR DESCRIPTION
**Pull request by salomao-rodrigues**

### CODE REVIEW

File: src/elements/text-input/package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "@emotion/core": "^10.0.10",
     "@uswitch/trustyle.frozen-input": "^0.1.3",
     "@uswitch/trustyle.styles": "^0.1.3",
-    "react": "^16.7.0"
+    "ramda": "^0.26.1",
```
Comment (annez) [inline src/elements/text-input/package.json]: Remove this please :)

Comment (salomao-rodrigues) [reply src/elements/text-input/package.json]: I might have forgotten -S when doing npm uninstall 🤦‍♂

File: package-lock.json [hunk 2]
```diff
@@ -7758,6 +7758,12 @@
             "path-is-absolute": "^1.0.0",
             "rimraf": "^2.2.8"
           }
+        },
```
Comment (annez) [inline package-lock.json]: It's still in the package lock @salomao-rodrigues :P

Comment (salomao-rodrigues) [reply package-lock.json]: I think that's a dependency of a dependency of storybook.
I got it removed in my last commit:
https://github.com/uswitch/trustyle/pull/3/commits/26ada1860b43d0746d28cb6d8f1e65029de1db63

Comment (annez) [reply package-lock.json]: haha

**Pull request merged**